svelte/tenantAdmin_organization.svelte

```diff
@@ -3,23 +3,239 @@
   import SideMenu from './_components/partials/SideMenu.svelte';
   import Navbar from './_components/partials/Navbar.svelte';
   import Footer from './_components/partials/Footer.svelte';
+  import { onMount } from 'svelte';
+  import { Title } from 'chart.js';
+  import { TenantAdminOrganization } from './jsApi.GEN';
+  import { notifier } from './_components/notifier';
 
   let segments = {/* segments */};
   let user = {/* user */};
+  let orgs = [/* orgs */];
+
+  let lastSent = {};
+  let elementIsDrag;
+
+  function renderTopLevelOrganization() {
```

Comment:
ini bukan cara svelte XD

kalo cara svelte bikin data structurenya dalam bentuk tree
lalu
{#each org
  {#each dept
     {#each div
        {#each job
     {#each job
  {#each job

Reply:
tapi gpp dah XD cuek, yg penting BE-nya bener